.konflux/scripts/render_snapshot.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import time
+
+
+def load_image_refs():
+    return json.loads(os.getenv("IMAGE_REFS", '[]'))
+
+
+def process_component(component, name_suffix):
+    if name_suffix != "":
+        name = f"{component['component']}-{name_suffix}"
+    else:
+        name = component["component"]
+    return {
+        "containerImage": component["ref"],
+        "name": name,
+        "source": {
+            "git": {
+                "revision": component["revision"],
+                "url": component["repository"]
+            }
+        }
+    }
+
+
+def construct_snapshot(
+    snapshot_name_prefix,
+    pipeline_run_name,
+    application,
+    components
+):
+    snapshot_name = f"{snapshot_name_prefix}-{int(time.time())}"
+    return {
+        "apiVersion": "appstudio.redhat.com/v1alpha1",
+        "kind": "Snapshot",
+        "metadata": {
+            "name": snapshot_name,
+            "labels": {
+                "appstudio.openshift.io/build-pipelinerun": pipeline_run_name
+            }
+        },
```

Comment:
Does it work correctly without providing the explicit `namespace`?

Reply:
Yes. The SA only knows about the `rh-acs-tenant` namespace.

Reply:
Magic. I feel that we may guard from some failures in the future if we wire up the actual namespace to be provided there directly. OTOH, I realize you'd need to do some extra setup for that so I'd leave it up to you to decide.

Reply:
Done in https://github.com/stackrox/stackrox/pull/13577/commits/9c3bbbb39d43450ad7679b348ee1c17e7117d973

It takes the namespace of the pipeline run - ie if we ever moved to a different namespace, it would update automatically.